frontend/packages/ux-editor/src/testing/componentSchemaMocks.ts

```diff
@@ -1,3 +1,6 @@
+import AlertSchema from './schemas/json/component/Alert.schema.v1.json';
+// import AccordionSchema from './schemas/json/component/Accordion.schema.v1.json';
+// import AccordionGroupSchema from './schemas/json/component/AccordionGroup.schema.v1.json';
```

Comment:
Could we delete these lines, or should we add a comment about why they are commented out and not deleted?